src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -2971,6 +2971,10 @@ void writeEvent(string ownerPrefix)
 
 					var eventSource = ownerPrefix.HasValue() ? ownerPrefix : "this";
 
+					// x:Bind to second-level method generates invalid code
+					// sanitizing member.Value so that "ViewModel.SearchBreeds" becomes "ViewModel_SearchBreeds"
+					var sanitizedMemberValue = SanitizeResourceName(member.Value.ToString());
```

Comment:
Looks like there's a potential null reference here, a member may not have a value if it has Objects:

https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=20975&view=logs&j=f37a9ee9-c1af-5a01-e40c-f9c310927886&t=1c94c882-7543-5b91-b2c9-7ef1343fcb4a&l=18959

Reply:
I see. I struggled to see how I could fully test this locally - is there a trick I am missing for that? I will move this assignment to be closer to where it is used and ensure it is suitably guarded (although I suspect it will be already otherwise the other code would throw!)

Reply:
In this case, running all the `Uno.UI.Tests` tests locally should validate this error: 

```
Object reference not set to an instance of an object. ---> System.InvalidOperationException: 
An error occurred when processing Grid at line 9:3 (Windows_UI_Xaml_Data\xBindTests\Controls\Binding_Event.xaml).
```